docs-ref-services/preview/resourcemanager-monitor-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager monitor client library for Java
 keywords: Azure, java, SDK, API, azure-resourcemanager-monitor, monitor
-author: ramya-rao-a
-ms.author: ramyar
+description: Documentation for the Azure Resource Manager monitor client library for Java - Version 2.0.0-beta.4.
```

Comment:
is it a dash or a period in Java - Version 2.0.0-beta.4.   ?

Reply:
I don't know. I copied and pasted it from the copy. That's how it looks.